docs/source/models/supported_models.rst

```diff
@@ -120,7 +120,7 @@ Text Generation (``--task generate``)
     - ✅︎
   * - :code:`CohereForCausalLM`
```

Comment:
```suggestion
  * - :code:`CohereForCausalLM`, :code:`Cohere2ForCausalLM`
```

Reply:
I thought the code meant the vllm classname (of which I did not make a new one for v2). If it is the architecture_name from the config then it makes sense.

Reply:
It should be the architecture name on HF side (so people can check the `config.json` and search for it in this list)